### ISSUE TITLE
SL Connection tab progressbar does not move while logging in to SL

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Animeimp/slmiv#1]
```
When logging in to SL via the SL Connection tab, the progressbar does not 
move.
```

Original issue reported on code.google.com by `joesox` on 18 Nov 2007 at 6:00

**Comment by GoogleCodeExporter** [comment]
```
Sometimes logging in to SL takes 30 to 90 seconds just like the real 
application. 
You will be forwarded to the 'My Inventory' tab when the import of inventory is 
complete.
```

Original comment by `joesox` on 18 Nov 2007 at 5:23

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 2.7.0
```

Original comment by `joesox` on 23 Nov 2007 at 12:31
- Changed state: **Verified**
